src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -86,6 +89,33 @@
 	 */
 	public static final long TRANS_SIZE = Long.MAX_VALUE - Integer.MAX_VALUE - 1;
 
+	/**
+	 * Containers that can represent audio or video media is by default
+	 * considered to be video. This {@link Map} maps such containers to the type
+	 * to use if they represent audio media.
+	 */
+	protected static final Map<String, AudioVariantInfo> audioOrVideoContainers;
+
+	static {
+		Map<String, AudioVariantInfo> mutableAudioOrVideoContainers = new HashMap<String, AudioVariantInfo>();
+
+		// Map container formats to their "audio variant".
+		mutableAudioOrVideoContainers.put(FormatConfiguration.MP4, new AudioVariantInfo(new M4A(), FormatConfiguration.M4A));
+		mutableAudioOrVideoContainers.put(FormatConfiguration.MKV, new AudioVariantInfo(new MKA(), FormatConfiguration.MKA));
+		mutableAudioOrVideoContainers.put(FormatConfiguration.OGG, new AudioVariantInfo(new OGG(), FormatConfiguration.OGA));
```

Comment:
@Sami32 That can be done later, it will be additional functionality. What I don't want is for PRs the remove existing functionality, so I think adding Flash in a later PR is fine.